spec/fixtures/fales/bloch.xml

```diff
@@ -79,6 +79,7 @@
             <genreform source="aat">Postcards.</genreform>
             <genreform source="aat">Posters.</genreform>
             <subject source="lcsh">Zines.</subject>
+            <subject source="lcsh">World War ||</subject>
```

Comment:
@ekate The acceptance criteria says "World War II" with capital i's, but it looks like these are bar characters instead. Is there a reason you used these characters?